python/tvm/tir/tensor_intrin/hexagon.py

```diff
@@ -163,3 +204,27 @@ def dot_product_32x2_i16i16i32_vdmpy(a: T.handle, b: T.handle, c: T.handle) -> N
 
 VRMPY_u8i8i32_VTCM_INTRIN = "dot_32x4_u8i8i32_vtcm_vrmpy"
 TensorIntrin.register(VRMPY_u8i8i32_VTCM_INTRIN, *generate_dot_product_32x4_u8i8i32("global.vtcm"))
+
+DMA_READ_1_u8 = "dma_read_1_u8"
```

Comment:
I don't see users for most of these.  Seems like it might be better to delete and allow users to create what is needed based on the test case or schedule?

Reply:
Good point, removed ✅